dotnet/samples/Concepts/Functions/MethodFunctions_Yaml.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Remember to also update the overview markdown file that references all the samples.

Reply:
Creating a follow up task for this: https://github.com/microsoft/semantic-kernel/issues/6690 as it's not a trivial thing to do i.e. we would have to make sure it's robust enough to handle mismatches etc.